frontend/src/index.css

```diff
@@ -13,7 +13,7 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   overflow-y: scroll !important; /* Show vertical scrollbar */
-  width: calc(100% - 14px) !important;
```

Comment:
I added this to keep the layout from getting broken in Chrome linux. I believe keeping it would be a better choice.

Reply:
hmm, in chrome macos it was just rendering a white block at the right of the screen which looks very bad. All html document seems like it has right margin of some value. But we can keep it if you want so badly.

Reply:
hmm weird, because in chrome linux there is no white block. Seems like it is a spit-down-beard / spit-up-moustache situation. I will approve the merge since mac-os users are numerous compared to linux users.

![](https://cdn-media-1.freecodecamp.org/images/0*erAJ6qLiiJi4B-wH.png)